src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs

```diff
@@ -69,7 +69,7 @@ internal static class CsvAppFileIO {
             Interlocked.Exchange(ref _busy, 0);
         }
 
-        private static async Task CreateCsvAndStartProcess(IREvaluationResultInfo result, IRSession session, string fileName, IFileSystem fileSystem, IProcessServices processServices) {
+        private static async Task CreateCsvAndStartProcess(IREvaluationResultInfo result, IRSession session, string fileName, IFileSystem fileSystem, IProcessServices processServices, IVsStatusbar statusBar) {
```

Comment:
Do you have a screenshot with a status bar? Just to check how it looks in conjunction with our broker dropdown